src/olympia/devhub/templates/devhub/addons/submit/done.html

```diff
@@ -95,7 +95,7 @@ <h3>{{ _("Version Signed") }}</h3>
   </p>
 {% endif %}
 <p>
-  <a class="button" href="{{ url('devhub.addons') }}">
+  <a class="button" href="{{ url('devhub.themes') }}">
```

Comment:
We only need static themes to go the themes url - extensions and other non-themes have to continue to go to `'devhub.addons'`.  So it will need wrapping in a `{% if addon.type == amo.ADDON_STATICTHEME %} ... {% else %} ... {% endif %}`